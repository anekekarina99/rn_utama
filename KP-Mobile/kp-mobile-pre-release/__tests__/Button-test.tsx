import React from 'react';
import {create} from 'react-test-renderer';
import {Button} from '@components/atoms/Button';

const tree = create(
  <Button
    label="Lanjut"
    background="#CED4DA"
    color="#868E96"
    icon=""
    width=""
  />,
);

test('snapshot', () => {
  expect(tree).toMatchSnapshot();
});
