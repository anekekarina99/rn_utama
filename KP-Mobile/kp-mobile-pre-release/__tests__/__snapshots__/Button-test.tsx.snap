// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#CED4DA",
      "borderColor": "#C2185B",
      "borderRadius": 20,
      "flex": 1,
      "justifyContent": "center",
      "opacity": 1,
    }
  }
>
  <Text
    style={
      [
        {
          "alignItems": "center",
          "flex": 1,
          "fontFamily": "Poppins",
          "fontSize": 16,
          "fontWeight": "600",
          "justifyContent": "center",
          "paddingBottom": 10,
          "paddingTop": 10,
        },
        {
          "color": "#868E96",
        },
      ]
    }
  >
    Lanjut
  </Text>
</View>
`;
